### PR TITLE
Nanotrasen Mimir-Class Rehabilitation (Prison) Vessel

### PR DESCRIPTION
**Pull request by Latentish**
## About The Pull Request
Adds the Mimir-Class Rehabilitation (Prison) Vessel
Removes the Tartarus
It's meant to simulate perma and the gulag, mixed with a bunch of maint stuff and low security staff. Starts with 3 security and 1 patient/prisoner.

mimirreal

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
The tartarus was our security/perma ship, so I want something better.

## Changelog

:cl:
add: Adds a mimir (sleepy)
code: adds some supporting code for the mimir (sleepy)
/:cl:

### CODE REVIEW

**Comment by Erikafox** [comment]
please bundle the tart's removal in with this, afaik it's still purchasable.

File: code/game/MapData/shuttles/nanotrasen_mimir.dm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/obj/item/storage/backpack/satchel/flat/mimir_transfer/PopulateContents()
+	new /obj/item/reagent_containers/food/drinks/bottle/whiskey(src)
+	new /obj/item/weldingtool/mini(src)
+	new /obj/item/choice_beacon/music(src)
+	new /obj/item/coin/antagtoken(src)
+
+//toilet =)
+
+/obj/structure/toilet/secret/mimir
+	secret_type = "/obj/item/storage/backpack/satchel/flat/mimir_transfer"
```
Comment (ZephyrTFA) [inline code/game/MapData/shuttles/nanotrasen_mimir.dm]: This probably isn't meant to be a text path

Comment (Latentish) [reply code/game/MapData/shuttles/nanotrasen_mimir.dm]: fixod

**Comment by Erikafox** [comment]
dreamseeker_eNk6mP5PJh
spellcheck
dreamseeker_VtA3Z5Om2Z
you can pixelshift these, I would, having it in the middle feels off
dreamseeker_PSlxFmOmtl
ditto
image
decal layering issue on the stand clear, not sure if I really like the orange techmaint, it takes away from the natural maint look.
dreamseeker_3x26CwJmRK
I'd take the techmaint out or make it run with the railings here
dreamseeker_vxqOFseAjt
td0vOwXa7F
wrong path
dreamseeker_XRl03TcZma
I would move the APC - Air alarm to share spare with the vending machines, then move the lockers over.
dreamseeker_PZJarYY3KQ
window shutters please

More review later.

File: _maps/configs/nanotrasen_mimir.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NTSV",
+	"namelists": ["GENERAL", "NATURAL_AGGRESSIVE", "BEASTS"],
+	"map_name": "Nanotrasen Mimir-Class Rehabillitation Vessel",
```
Comment (Erikafox) [inline _maps/configs/nanotrasen_mimir.json]: Unless the Nanotrasen is intentional, I'd cut it.

File: _maps/configs/nanotrasen_mimir.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+			"outfit": "/datum/outfit/job/security",
+			"slots": 2
+		},
+		"Facility Physician": {
+			"outfit": "/datum/outfit/job/brig_phys",
+			"slots": 1
+		},
+		"Prisoner": {
+			"outfit": "/datum/outfit/job/prisoner",
+			"slots": 5
```
Comment (Erikafox) [inline _maps/configs/nanotrasen_mimir.json]: you talked about changing the rest of these numbers up, leaving a comment as a reminder.

File: _maps/configs/nanotrasen_mimir.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+		}
+	},
+	"enabled": true
```
Comment (Erikafox) [inline _maps/configs/nanotrasen_mimir.json:28]: depending on how this runs, I may prefer this ship be adminspawn.

**Review by Erikafox — COMMENTED** [review]
not a comprehensive, just a few things while I'm thinking

**Comment by Zevotech** [comment]
5 (or more) prisoners can easily end up overpowering the sec staff if allowed any kind of freedom, especially in accessing ruins and their loot. Might need to lower it so that having a lot of prisoners is the ship owners choice when adding slots.

**Comment by Latentish** [comment]
> 5 (or more) prisoners can easily end up overpowering the sec staff if allowed any kind of freedom, especially in accessing ruins and their loot. Might need to lower it so that having a lot of prisoners is the ship owners choice when adding slots.
I like this idea a lot.

**Review by Erikafox — COMMENTED** [review]
image
I would undo the silly wiring here. It doesn't really mesh with the ordered techmaint.
StrongDMM_EhUd7JNh6P
I'd get some wireknots under the windows to the prison.
StrongDMM_4owwX2qjQ8
little wiring nitpick, I'd use a 90 degree turn instead of the diag, since there's something on the diag.
StrongDMM_oBnNQNsICb
I'd pixelshift the button to the edge of the wall, or maybe even put it on the outer edge (top of the image) of the hull. 
Also not huge on having to climb railings to screw with prechargers, try a windoor in the middle maybe?
If you are insistent on keeping the orange techmaint pattern, I would like the tiles the railing is on to be plating. 
StrongDMM_2Pk5jcrITT
Kinda like such.
StrongDMM_V3QcqoiWAr
This space is too cramped for what it does, they also need an autolathe. 
Ditch the holofan projector too imo.
StrongDMM_ZzrOHjD2tw
Give the bridge door an access lock, even if it's a gameplay concession I don't think it really makes sense.
StrongDMM_tU9UhIwmLo
I'm not saying you should subtype a chemsprayer to start filled with pepperspray and give it to the sec team. But that would be incredibly funny. Also move the tear gas and awards to the warden's box. 
StrongDMM_PQEOGDaJ4P
Move the stimpacks to sec, leave a note saying they're for the prisoners when they're out mining, give them standard emergency oxygen tanks instead of the double yellows. I don't think I've ever run out of air on a double yellow and I've played entire rounds in a gas mask
StrongDMM_BbyZL293Dg
Move the AAC down a few pixels, put a second vent inside so it cycles faster. 
Put some rusted reinforced walls around, there's interior rust but not enough
StrongDMM_jjDwQVtjlP
This is so fucking illegal. At least use a falsewall that's connected the room. 
StrongDMM_Gq6kviaTr2
Give em some athletic shorts too
StrongDMM_2FOrEpkzjq
FYI you can give this dood some custom lines via var editing. 
StrongDMM_L6i9ZeFDUo
layering
StrongDMM_ftOLVEuC6u
try plating, or grey techmaint under conveyors. 
StrongDMM_PhXbMR0Lpw
I'm actually begging you to not use diag wires under false walls
The Central Hallway and Brig Cells need to have at least another scrubber + vent. I'd maybe do the same in Dormitory 3
StrongDMM_exK9jxINlR
See this?
StrongDMM_gmAn1nhOQ6
Try this instead.
StrongDMM_mOILKsqHQh
these'll layer weird, put the directional windows on the outside of the tiles instead.
StrongDMM_2O1KUBvoZW

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Latentish** [comment]
guh guh guh me want see round but not time also not respond to advice yet ooga booga

**Comment by Latentish** [comment]
I'm gonna draft this for now

**Comment by Latentish** [comment]
Last commit also made the bridge door sec locked.

**Comment by Latentish** [comment]
And adds an autolathe.

**Comment by Erikafox** [comment]
The ship played fine last time I saw it but I'd still like to talk about making it a-spawn

**Comment by Daved27hundred** [comment]
needs another pacman because with the recent change to how ships are piloted this thing guzzles power like crazy, and one pacman isn't enough.

**Comment by Zevotech** [comment]
replace the rolling table in med because of the bug that makes it leave things behind. Lost the surgery bag once to that, wasn't fun.

**Comment by Erikafox** [comment]
image
image
window shutters on both exterior windows
image
make the pump to the air recycler, and to one of the airtanks start on.
firefox_2UFi3DA6jA
change the x on these to -28

**Review by Erikafox — APPROVED** [review]

**Review by Erikafox — CHANGES_REQUESTED** [review]
This needs a stationary docking port on the exterior. Ping me in ruinyard or look at the Borea for an example

**Review by Erikafox — APPROVED** [review]

**Pull request merged**